### ISSUE TITLE
Deploy flexible csv and New Fields

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/MyopicVicar#1472]
Depends on #1473

**Comment by Captainkirkdawson** [comment]
flexible csv and new fields deployed to production

**Comment by SteveBiggs** [comment]
We'll need to decide on a communucation plan - bearing in mind that ony spreadsheet users can currently use it, not WinREG.

**Comment by Captainkirkdawson** [comment]
@SteveBiggs put your comment into new story #1537

**Comment by SteveBiggs** [comment]
Thanks @Captainkirkdawson

**Comment by richpomfret** [comment]
Deployed.
